common/data/dotdigital.ts

```diff
@@ -0,0 +1,36 @@
+type Campaign = {
+  id: number;
+  slug: string;
+  label: string;
+  description?: string;
+};
+
+export const newsletterCampaign: Campaign = {
```

Comment:
This lives in the same folder as the "microcopy" file, and matches the terminology used by Dotdigital.